VideoLocker/res/values/errors.xml

```diff
@@ -10,10 +13,14 @@
     <string name="reset_no_network_title">Connection Error</string>
     <!-- Alert dialog content shown when a network request fails because the user has no internet connection -->
     <string name="reset_no_network_message">You are not connected to the Internet. Please check your Internet connection.</string>
```

Comment:
Not sure why the "reset" prefix is here in these string names. I think it should be replaced with "dialog".